r/R/dataset-scan.R

```diff
@@ -56,7 +56,11 @@
 Scanner <- R6Class("Scanner", inherit = ArrowObject,
   public = list(
     ToTable = function() dataset___Scanner__ToTable(self),
-    Scan = function() dataset___Scanner__Scan(self)
+    ScanBatches = function() dataset___Scanner__ScanBatches(self),
+    Scan = function() {
+        .Deprecated("ScanBatches")
```

Comment:
This is nice but probably not necessary, this was a (de facto) internal method. If it were me I'd just delete it, but since you've already done this, might as well keep it. Can you please just make a JIRA to delete this so we don't forget to clean this up after the release?

Reply:
We've been using ARROW-11782 so I'll note that in this PR and update the issue.